apps/showcase/src/components/training/code-editor-view/code-editor-view.component.ts

```diff
@@ -195,7 +195,9 @@ export class CodeEditorViewComponent implements OnDestroy {
    */
   private readonly monacoPromise = firstValueFrom(this.newMonacoEditorCreated.pipe(
     map(() => window.monaco)
-  ));
+  )).catch(
+    (err) => console.error(err)
```

Comment:
Should not use the logger instead?

Reply:
I don't think we set a logger for now on the Otter showcase 
If we don't have one set up, I would expect to do it in another PR

Reply:
shouldn't we let it `throw`? Lot of things will be broken if we don't have the global `monaco` instance

Reply:
It will throw if the monaco editor is never instantiated and the newMonacoEditorCreated subject is completed.
This is the case in the unit tests. 
I could catch the monacoPromise in the tests themselves but that would require making it public and I prefer adding a catch than transforming a private property into a public one for testing purposes.

Reply:
we should be able to test monaco-editor in the unit-tests? only the webcontainer part should be mocked?